lib/ansible/galaxy/collection.py

```diff
@@ -1119,6 +1298,32 @@ def _get_collection_info(dep_map, existing_collections, collection, requirement,
     dep_map[to_text(collection_info)] = collection_info
 
 
+def parse_scm(collection, version):
+    if ',' in collection:
```

Comment:
@s-hertel why use comma? It's quite unconventional. Is `@` already taken by something else?

Reply:
It's the syntax supported by roles and was just there for similarity.

Reply:
Ah, I see. I haven't used it so it looks weird based on the experience with other ecosystems. But in this case, consistency makes sense.